mllib-local/src/main/scala/org/apache/spark/ml/linalg/BLAS.scala

```diff
@@ -244,6 +244,15 @@ private[spark] object BLAS extends Serializable {
   }
 
   /**
+   * y += alpha * A * x
+   *
+   * @param A The upper triangular part of A in a [[DenseVector]] (column major)
+   */
+  def dspmv(n: Int, alpha: Double, A: DenseVector, x: DenseVector, y: DenseVector): Unit = {
```

Comment:
Actually BLAS DSPMV performs matrix-vector operation ```y := alpha*A*x + beta*y```, should we provide the same function as original BLAS? Since it would be used by other MLlib function in the future.